Makefile

```diff
@@ -83,16 +83,21 @@ TAR_XFORM_FLAG = $(shell $(TAR) --version | grep -q GNU && echo "--xform=s" || e
 
 GIT_DIR := $(shell git rev-parse --git-dir 2> /dev/null)
 
+# We intentionally use LINKFLAGS instead of the more traditional LDFLAGS
```

Comment:
can this be up near the other flags? this one may conceivably be overridden on the command line or in the environment. for now, we can put it in the command-line only section.